src/main/java/com/hbm/blocks/ModBlocks.java

```diff
@@ -3200,6 +3204,7 @@ private static void registerBlock() {
 		GameRegistry.registerBlock(machine_forcefield, machine_forcefield.getUnlocalizedName());
 		GameRegistry.registerBlock(radiorec, radiorec.getUnlocalizedName());
 		GameRegistry.registerBlock(radiobox, radiobox.getUnlocalizedName());
+		GameRegistry.registerBlock(fluid_pump, fluid_pump.getUnlocalizedName());

```

Comment:
You should use `register(fluid_pump)` here, not only is it shorter but it also automatically sets the ItemBlock to `ItemBlockBase.class` which is necessary for the tooltip to work.